components/gif_picker/components/InfiniteScroll/index.js

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+export default class InfiniteScroll extends Component {
+    static propTypes = {
+        children: PropTypes.array,
+        element: PropTypes.string,
+        hasMore: PropTypes.bool,
+        initialLoad: PropTypes.bool,
+        loader: PropTypes.object,
+        loadMore: PropTypes.func.isRequired,
+        pageStart: PropTypes.number,
+        threshold: PropTypes.number,
+        useWindow: PropTypes.bool,
+        isReverse: PropTypes.bool,
+        containerHeight: PropTypes.number,
+        scrollPosition: PropTypes.number,
+    }
+
+    static defaultProps = {
+        element: 'div',
+        hasMore: false,
+        initialLoad: true,
+        pageStart: 0,
+        threshold: 250,
+        useWindow: true,
+        isReverse: false,
+        containerHeight: null,
+        scrollPosition: null,
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.scrollListener = this.scrollListener.bind(this);
```

Comment:
Can you leverage an arrow function to obviate the need to `bind` and override the `constructor`?